### ISSUE TITLE
[feature-request] fn deploy --all ?

### ISSUE THREAD

**Issue by denismakogon** [fnproject/cli#280]
Would nice to support the building of entire application images. 
CLI already supports `fn deploy --all`, why not `fn build --all`?

**Comment by shaunsmith** [comment]
Agreed.  This is a missing feature.  And `fn push` should probably also support `--all`.
